psa_car_controller/psacc/application/battery_charge_curve.py

```diff
@@ -1,58 +1,58 @@
-from statistics import mean
```

Comment:
please correct the style to avoid to many changes in the PR

Reply:
Hi Florian,

I was off last week and couldn't come back on this earlier.
Note that this is my first contribution in github: I am not a professional developer and I am new to a lot of things: git / github, python, vs code, docker... I start learning all of it by myself a few month back when looking at Home Assistant and what can be done with it.
I have spent some time looking at your project to understand how it works and the issues I had with it. For my knowledge, but also to be able to help rather than simply flagging issues and waiting for other to resolve them.
And I want to thank you and congratulate you for all the work you have done: when looking at it, there is much more work and effort required to build it than I would have imagined at first.

To come back to the style issue: I have noticed it on github, but I am not sure where it is coming from or how to fix. When I look at my project in vsCode, it only flag my changes, but then in github all the file is highlighted. I have tried to restart from the master repo and redo the changes, but end up in the same way.
I suspect it may have to do with me starting on Windows first then moving to Linux/WSL and maybe issues with line/carriage return, but couldn't find what go wrong exactly. 
I would like to understand what goes wrong and fix that, if you have some tips they are welcome, if not I will check again when I can, but fell free also to fix it yourself if it helps going on: I had put a description of the change made for that.

Also, I have run the app with my fix for a while on my system. The fix helps and avoid the divide by zero issue, but I have found other problems. There is also a bug in the stellantis/PSA API, for my car at least, where sometime the autonomy is 0km and so battery empty, still the API returns battery level as 100%. I confirm this is incorrect info coming from PSA (also in their app), and not caused by your app. This however cause other issues with your app:
- mess up the recording and retrieving/displaying of info: I dind't have time to investigate it further yet;
- with charge control enabled, the app will stop the charge, even if the stop charge level is set to 100%, when it sees that the battery level is 100%, although in reality battery is empty. => I am setting the charge level to 101% as workaround: not your app's fault, but it can be improved to better manage such exceptional situation and I will add a separate feature request for that.

I think it can help you and the community to understand the various issues that are reported around that.

Reply:
Corrected now